apps/aecore/src/aec_keys.erl

```diff
@@ -492,6 +545,14 @@ p_gen_new_peer(Pwd, PubFile, PrivFile) ->
             error({generated_key_pair_check_failed, [PubKey, PrivKey]})
     end.
 
+ensure_dir(KeysDir) ->
```

Comment:
Will this work if also the parent directory is missing?

Reply:
Updated to use `filelib:ensure_dir/1` instead of `file:make_dir/1` ✅